src/NuGet/BuildNuGets.csx

```diff
@@ -87,6 +87,7 @@ string[] RedistPackageNames = {
     "Microsoft.CodeAnalysis.EditorFeatures",
     "Microsoft.CodeAnalysis.EditorFeatures.Text",
     "Microsoft.CodeAnalysis.Features",
+    "Microsoft.CodeAnalysis.Remote.Razor.ServiceHub",
```

Comment:
These assemblies are most similar/related to other packages that are in the redist list such as the VisualStudio language services packages which is why I started there.

It sounds like the only thing then is to determine what license is appropriate which I can't really answer, who is the right person to loop in?

We don't currently have a scenario defined where a 3rd party developer would include or depend upon this functionality in its current form.